tests/unit/activation/test_fish.py

```diff
@@ -21,3 +21,33 @@ def print_prompt(self):
             return "fish_prompt"
 
     activation_tester(Fish)
+
+
+@pytest.mark.skipif(IS_WIN, reason="we have not setup fish in CI yet")
+def test_fish_path(activation_tester_class, activation_tester, monkeypatch, tmp_path):
```

Comment:
Any reason we can't include this into test_fish?

Reply:
@gaborbernat I've moved the test into `test_fish`.